charts/kyverno-policies/values.yaml

```diff
@@ -1,56 +1,49 @@
-# Supported- baseline/restricted/privileged/custom
-# For more info- https://kyverno.io/policies/pod-security
+# -- Pod Security Standard profile (`baseline`, `restricted`, `privileged`, `custom`).
+# For more info https://kyverno.io/policies/pod-security.
 podSecurityStandard: baseline
-# Supported- low/medium/high
+
+# -- Pod Security Standard (`low`, `medium`, `high`).
 podSecuritySeverity: medium
-# Policies to include when podSecurityStandard is custom
+
+# -- Policies to include when `podSecurityStandard` is `custom`.
 podSecurityPolicies: []
-# Additional policies to include from "other"
-# Example:
-#   includeOtherPolicies:
-#   - require-non-root-groups
+
+# -- Additional policies to include from `other`.
 includeOtherPolicies: []
-# Supported values- `audit`, `enforce`
-# For more info- https://kyverno.io/docs/writing-policies/validate/
+# - require-non-root-groups
+
+# -- Validation failure action (`audit`, `enforce`).
+# For more info https://kyverno.io/docs/writing-policies/validate.
 validationFailureAction: audit
-# Define validationFailureActionOverrides for specific policies.
-# The overrides for 'all' will apply to all policies
-# Eg:
-# validationFailureActionOverrides:
-#   all:
-#     - action: audit
-#       namespaces:
-#         - ingress-nginx
-#   disallow-host-path:
-#     - action: audit
-#       namespaces:
-#         - fluent
+
+# -- Define validationFailureActionOverrides for specific policies.
+# The overrides for `all` will apply to all policies.
 validationFailureActionOverrides:
   all: []
-# Exclude resources from individual policies
-# Eg:
-# policyExclude:
-#   disallow-host-path:
-#     any:
-#     - resources:
-#         kinds:
-#         - Pod
-#         namespaces:
-#         - fluent
-#
-# Policies with multiple rules can have individual rules excluded:
```

Comment:
Would still be good to include some kind of text that `policyExclude` can done on whole policies or individual rules within a policy

Reply:
I added a note in the comment, but I will need to understand the syntax a little better to provide a useful example.

Reply:
Got it, the key in the map can be the policy name or the rule name. This could be confusing as rule names are not guaranteed to be unique.
We could either change the structure of the values or add a check in the chart that fails if it detects duplicate rule names across policies.